docs/advanced.rst

```diff
@@ -265,8 +256,12 @@ environment (or lmod) modules.
 Environment Variables
 ~~~~~~~~~~~~~~~~~~~~~
 
-You can set/unset environment variables in your test configs, and use
-them in your scripts.
+*Full Docs:* :ref:`tests.env.variables`
+
+You can set environment variables in your test scripts using the
+'env' section under both 'run' and 'build'. This will cause the variables to
+be exported within the generated run or build script, where they can be used
+by commands run as part of that script.
```

Comment:
```suggestion
'env' section under both 'run' and 'build'. This will cause the variables to
be exported within the generated run or build script, where they can be used
by commands run as part of that script. You may only reference environment
variables in the 'cmds' subsection section of the 'run' and 'build' sections,
as they are the only sections that are converted into shell scripts. 
```